index.html

```diff
@@ -91,6 +91,9 @@
 									<a href="examples/emberjs/" data-source="http://emberjs.com" data-content="Ember is a JavaScript framework for creating ambitious web applications that eliminates boilerplate and provides a standard application architecture.">Ember.js</a>
 								</li>
 								<li class="routing">
+									<a href="examples/openui5/" data-source="http://openui5.org" data-content="OpenUI5 lets you build enterprise-ready web applications, responsive to all devices, running on almost any browser of your choice.">OpenUI5</a>
+								</li>
+								<li class="routing">
```

Comment:
This is unrelated to the rename so maybe do this in a separated PR (or at least in a separated commit).